pure.zsh

```diff
@@ -248,18 +252,19 @@ prompt_pure_async_vcs_info() {
 	# to be used or configured as the user pleases.
 	zstyle ':vcs_info:*' enable git
 	zstyle ':vcs_info:*' use-simple true
-	# Only export two message variables from `vcs_info`.
-	zstyle ':vcs_info:*' max-exports 2
-	# Export branch (%b) and Git toplevel (%R).
+	# only export three msg variables from vcs_info
```

Comment:
Don't do unrelated changes.

Reply:
Done. Not sure how I did that, possibly started the PR from an older version of the code (I still changed "two" to "three")